plugins/woocommerce/tests/php/includes/data-stores/class-wc-customer-data-store-test.php

```diff
@@ -64,7 +65,7 @@ public function test_meta_data_cannot_overwrite_column_data() {
 	 * @return string[]
 	 */
 	public function get_pending_only_as_order_statuses() {
-		return array( 'wc-pending' => 'pending' );
+		return array( OrderInternalStatus::PENDING => 'pending' );
```

Comment:
I guess `'pending'` value here could be replaced with `OrderStatus::PENDING`?

Reply:
Good catch, @kalessil ! Replaced and also fixed all lint issues 👍